examples/knowledge_tuning/translation_example/flows/translate_flow_knowledge.yaml

```
@@ -0,0 +1,57 @@
+- block_type: LLMBlock
+  block_config:
+    block_name: question_generation
+    model_id: "ibm-granite/granite-3.3-2b-instruct"
+    config_path: configs/generate_question.yaml
+    output_cols:
+      - question
+    parser_kwargs:
+      parser_name: custom
+      parsing_pattern: (?:Question|QUESTION)\:\s*(.*)
+      parser_cleanup_tags:
+        - "[END]"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)